### PR TITLE
docs(README): standardize

### LINKED ISSUE #62: Standardize README

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
### Noteable Changes

[README](https://github.com/michael-ciniawsky/worker-loader/tree/docs)

### Issues

- Fixes #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,25 +139,46 @@ const worker: Worker = new MyWorker();
...
+          </br>
+          Joshua Wiens
         </a>
-        <br />
-        <a href="https://github.com/d3viant0ne">Joshua Wiens</a>
       </td>
       <td align="center">
-        <a href="https://github.com/TrySound">
-          <img width="150" height="150" src="https://avatars3.githubusercontent.com/u/5635476?v=3&s=150">
+        <a href="https://github.com/sapegin">
```
Comment (d3viant0ne) [inline README.md]: Per Tobias, max of 5. Artem works on defaults, he doesn't maintain loaders & plugins for the org.

**Comment by TrySound** [comment]
@michael-ciniawsky Of course. `getOptions` works for both cases.

File: README.md (hunk 1 above)
Comment (michael-ciniawsky) [inline README.md]: It's from the template, `Repo Maintainer(s)` + `Org Maintainers` agreed to slim it down a bit, but needs an update in ` defaults aswell then :)

**Comment by michael-ciniawsky** [comment]
It was more of being a recommended path (`.worker.js`) we should document or not :) kk I will update then `webpack.config.js` is recommended in general, so it better anyways

**Pull request merged**